### ISSUE TITLE
Post traumatic phenotype of the eye

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10931]
**Preferred term label:**
Post traumatic phenotype of the eye

**Synonyms**
na

**Definition (free text, please give PubMed ID)**
Useful as a parent term to classify the presence post traumatic phenotypes such as eyelid lacerations,  bulbus rupture, intraocular foreign bodies and so on.

**Parent term (use hpo.jax.org/app)**
HP:12372

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
NA

**Your nano-attribution (ORCID)**
0000-0001-9042-0508

**Comment by daboe01** [comment]
sorry for the dupe, i just learned about 6000711
